libenkf/src/enkf_state.c

```diff
@@ -794,7 +800,9 @@ int enkf_state_load_from_forward_model(enkf_state_type * enkf_state ,
   if (ensemble_config_have_forward_init( enkf_state->ensemble_config ))
     result |= enkf_state_forward_init( enkf_state , run_arg );
 
+  printf("AFter forward:%d \n",result);
   result |= enkf_state_internalize_results( enkf_state , run_arg , msg_list );
+  printf("AFter internalize:%d \n",result);
```

Comment:
Why are the Fs capitalized?

Reply:
This is debug output - it should go entirely.